webcompat/templates/contributors.html

```diff
@@ -3,7 +3,7 @@
 {% block body %}
 <div>
   {% include "nav/nav.html" %}
-  <main role="main">
```

Comment:
random note that `<main role="main">` (here and in the other cases) is still necessary for IE11 compat (as it doesn't know, so doesn't expose, the main landmark role)